.github/ISSUE_TEMPLATE/decommissioning_v_1_0.md

```diff
@@ -0,0 +1,58 @@
+# Decommissioning a Linux Server  
+
+**Describe wich and why will be a server decommissioning**
+<p>ID-Service Now: <input type="text" size="20" maxlength="40" name="ID-Number" value="ID-Number" /></p>  <!-- put the ID Number here -->
+<!-- A clear and concise description of what the problem is. -->  
+
+**Screenshots**
+<!-- If applicable, add screenshots to help explain your problem. -->
+
+**Additional context**
+<!-- Add any other context about the problem here. -->
+
+
+# Workflow
+
+<!--The steps from confluence https://unibe-ch.atlassian.net/wiki/x/moU8Aw  are followed.  -->
+
+>**Servername :** <!-- Servername to decommission -->  
+>**FQDN:** <!-- FQDN -->  
+>**IP:** <!-- IP -->  
+>**Contact person:** <!-- Contact person -->  
+>**Projectname:** <!-- Name of the project -->  
```

Comment:
I'd put these system facts in the first general information part above (directly after the first heading?) and only leave action items in this section.